mongomock/aggregate.py

```diff
@@ -5,6 +5,8 @@
 import copy
 import datetime
 import itertools
+import warnings
```

Comment:
Please move after the other std library imports (after random but before six), and move the six imports (third party) as a different import group.